packages/platform-sdk-ark/src/dto/transaction.ts

```diff
@@ -16,7 +18,7 @@ export class TransactionData extends DTO.AbstractTransactionData implements Cont
 	}
 
 	public confirmations(): BigNumber {
-		return BigNumber.make(this.data.confirmations);
+		return bigNumber(this.data.confirmations);
```

Comment:
confirmations don't have decimals, but amounts do. how do we differentiate the two?

Reply:
I did a mass search-and-replace, fixing those at the moment.